Project/MyGameLibrary/Character/Character.cs

```diff
@@ -7,26 +7,27 @@ public class Character
     {
         public string CharacterName { get; set; }
         public bool IsDead { get; set; }
-        private int LoveScore;
-        public int GetLoveScore() {
-            return LoveScore;
-        }
-        public void SetLoveScore(int LScore) {
-            if (IsDead == true) {
-                LoveScore = -1;
-                return;
+        public int LoveScore {
+            get {
+                return LoveScore;
```

Comment:
Make sure you test all of this and this works.


Also, fix naming violations. 

The getter setter variable should be LoveScore and the internal private variable should be _LoveScore or something like that, please use naming conventions.